bedrock/firefox/templates/firefox/australis/whatsnew.html

```diff
@@ -55,10 +55,10 @@
 {% block aside_content %}
   <section class="learn-more billboard">
     <ul>
-      <li><a href="https://support.mozilla.org/products/firefox" target="_blank" title="{{ _('Opens new window') }}">{{ _('Need help?') }}</a></li>
-      <li><a href="https://support.mozilla.org/kb/learn-more-about-the-design-of-new-firefox" target="_blank" title="{{ _('Opens new window') }}">{{ _('Learn about the new design') }}</a></li>
-      <li><a href="https://support.mozilla.org/kb/what-happened-to-the-add-on-bar" target="_blank" title="{{ _('Opens new window') }}">{{ _('Where did my add-on bar go?') }}</a></li>
-      <li><a href="https://support.mozilla.org/kb/common-questions-after-updating-to-new-firefox" target="_blank" title="{{ _('Opens new window') }}">{{ _('Frequently asked questions') }}</a></li>
+      <li><a href="https://support.mozilla.org/products/firefox" target="_blank" rel="noopener noreferrer" title="{{ _('Opens new window') }}">{{ _('Need help?') }}</a></li>
```

Comment:
I don't think we need `target="_blank"` on these links anymore. Please remove, along with `title="{{ _('Opens new window') }}"`.